tests/torch/data/reference_graphs/quantized/hawq/inception_v3_device_VPU__mode_liberal__ratio_1.5__with_target_scopes.dot

```diff
@@ -1,730 +0,0 @@
-strict digraph  {
```

Comment:
@ljaljushkin note that due to our policy of avoiding requantizations whenever possible, the extension of avg_pool2d allowed quantizer configs led to a new quantization scheme of inception_v3, where the branch quantizers that were previously left unmerged due to unresolvable qconfig conflicts are now merged:
![Screenshot from 2021-06-12 16-18-12](https://user-images.githubusercontent.com/31695470/121777468-ec9c4b00-cb9a-11eb-9eaf-76e72ef6a515.png)

However, since only the 8-bit options are common among the branch AFQs, the resulting merged quantizer only has 8-bit quantizations allowed, which means that the 4-bit weight-quantizers sinking the 8-bit activations would still be considered as 8-bit operations, with corresponding bit complexity, which ultimately means that a lower compression ratio could be achieved overall. The max ration with the new config seems to be 1.44, and I had to lower the tested ratio further to 1.3 to test a mix of the various WFQ bitwidths rather than a uniform 4-bit WFQ distribution.

Reply:
@vuiseng9